doc/node_config_json_schema.json

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
Nit: we ought to have some way to comment in here that this is a generated file that should not be edited manually.

Reply:
If only was a jsonc 😅

Reply:
not sure how, if we had jsonc it would be doable

Reply:
`"//": "this is autogenerated"` :-p

Reply:
yes so we can see changes in the schema when someone adds a flag.
`"//": "this is autogenerated"` will pop in the json schema for users which isnt nice 😢